<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-33C0BU9" failures="6" tests="42" name="api.tests.datadriven.users.postTests" time="71.047" errors="0" timestamp="2024-09-08T22:26:42 IST" skipped="0">
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="1.276"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="1.230"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="1.340"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="1.598"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="1.559"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod1" time="4.747"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.563"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.470"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.465"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.798"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.591"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod2" time="1.699"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.408"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.430"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.329"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.536"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.362"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod3" time="1.435"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.838"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.782"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.525"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.840"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.449"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod4" time="1.883"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="1.436"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="2.168"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="1.940"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="1.621"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="2.047"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod5" time="1.297"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="1.988">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="1.685">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="2.384">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="1.994">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="1.628">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod6" time="1.466">
    <failure message="
Expected: is a value greater than &lt;6&gt;
     but: &lt;4&gt; was less than &lt;6&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
Expected: is a value greater than <6>
     but: <4> was less than <6>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at api.tests.datadriven.users.postTests.PostUserTestMethod6(postTests.java:184)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- PostUserTestMethod6 -->
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.584"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.637"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.809"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.324"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.413"/>
  <testcase classname="api.tests.datadriven.users.postTests" name="PostUserTestMethod7" time="1.473"/>
</testsuite> <!-- api.tests.datadriven.users.postTests -->
